src/mockData/fakeMovies.js

```diff
@@ -1,14 +1,14 @@
 import characterResults from './fakeCharacters';
 
 const movieResults = [
-  {episode_id: 1, title: "the fast and the furious", year: "1999-07-05", scroll: "words words words", characters: characterResults},
-  {episode_id: 2, title: "2 fast 2 furious", year: "2000-07-05", scroll: "words words words", characters: characterResults},
-  {episode_id: 3, title: "Tokyo Drift", year: "2001-07-05", scroll: "words words words", characters: characterResults},
-  {episode_id: 4, title: "fast & furious", year: "2002-07-05", scroll: "words words words", characters: characterResults},
-  {episode_id: 5, title: "fast 5", year: "2003-07-05", scroll: "words words words", characters: characterResults},
-  {episode_id: 6, title: "furious 6", year: "2004-07-05", scroll: "words words words", characters: characterResults},
-  {episode_id: 7, title: "furious 7", year: "2005-07-05", scroll: "words words words", characters: characterResults},
-  {episode_id: 8, title: "the f8 of the furious", year: "2006-07-05", scroll: "words words words", characters: characterResults}
+  {episode_id: 1, title: "the fast and the furious", release_date: "1999-07-05", scroll: "words words words", characters: characterResults, opening_crawl: "paaaam-pam paaaaam-pam pa-pa-paaaam"},
+  {episode_id: 2, title: "2 fast 2 furious", release_date: "2000-07-05", scroll: "words words words", characters: characterResults, opening_crawl: "paaaam-pam paaaaam-pam pa-pa-paaaam"},
+  {episode_id: 3, title: "Tokyo Drift", release_date: "2001-07-05", scroll: "words words words", characters: characterResults, opening_crawl: "paaaam-pam paaaaam-pam pa-pa-paaaam"},
```

Comment:
Ah shoot, I forgot to take "scroll" out of this.  I will do it on my next update.